src/modules/string/StringTest.cpp

```diff
@@ -354,7 +354,7 @@ TEST_F(StringTest, shouldGenerateBinary)
 {
     const auto binaryLength = 8;
 
-    const auto binary = String::binary(binaryLength);
+    const auto binary = String::binary({}, binaryLength);
```

Comment:
Could you add some test for checking if it generated at least x characters and at most y characters?. Also I would be nice to have a test when the guarantee is broken (for example at least characters amount is greater than length).

Reply:
I added some tests, I think we can add a lot more but I am not that familiar with unit testing. Let me know how I can improve the added tests